[2024-07-30T10:28:21.446+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-30T10:28:21.496+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipelinev5.load_dimension_tables.load_users_dim_table scheduled__2018-11-29T00:00:00+00:00 [queued]>
[2024-07-30T10:28:21.518+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipelinev5.load_dimension_tables.load_users_dim_table scheduled__2018-11-29T00:00:00+00:00 [queued]>
[2024-07-30T10:28:21.519+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-30T10:28:21.649+0000] {taskinstance.py:2330} INFO - Executing <Task(LoadDimensionOperator): load_dimension_tables.load_users_dim_table> on 2018-11-29 00:00:00+00:00
[2024-07-30T10:28:21.660+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1398) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-07-30T10:28:21.662+0000] {standard_task_runner.py:63} INFO - Started process 1402 to run task
[2024-07-30T10:28:21.660+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pipelinev5', 'load_dimension_tables.load_users_dim_table', 'scheduled__2018-11-29T00:00:00+00:00', '--job-id', '292', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_dag.py', '--cfg-path', '/tmp/tmpf2ic87ku']
[2024-07-30T10:28:21.663+0000] {standard_task_runner.py:91} INFO - Job 292: Subtask load_dimension_tables.load_users_dim_table
[2024-07-30T10:28:21.807+0000] {task_command.py:426} INFO - Running <TaskInstance: pipelinev5.load_dimension_tables.load_users_dim_table scheduled__2018-11-29T00:00:00+00:00 [running]> on host c3b60429ae20
[2024-07-30T10:28:22.228+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pipelinev5' AIRFLOW_CTX_TASK_ID='load_dimension_tables.load_users_dim_table' AIRFLOW_CTX_EXECUTION_DATE='2018-11-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2018-11-29T00:00:00+00:00'
[2024-07-30T10:28:22.230+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-30T10:28:22.277+0000] {load_dim_operator.py:26} INFO - Loading dimension table users
[2024-07-30T10:28:22.317+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:28:22.379+0000] {sql.py:487} INFO - Running statement: 
                BEGIN;
                TRUNCATE TABLE users;
                
        INSERT INTO users (user_id, first_name, last_name, gender, level)
        SELECT DISTINCT ON (userid) userid, firstname, lastname, gender, level
        FROM staging_events
        WHERE page='NextSong'
        ON CONFLICT (user_id) DO UPDATE
        SET first_name = EXCLUDED.first_name,
            last_name = EXCLUDED.last_name,
            gender = EXCLUDED.gender,
            level = EXCLUDED.level
    ;
                COMMIT;
            , parameters: None
[2024-07-30T10:28:22.921+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-30T10:28:22.942+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pipelinev5, task_id=load_dimension_tables.load_users_dim_table, run_id=scheduled__2018-11-29T00:00:00+00:00, execution_date=20181129T000000, start_date=20240730T102821, end_date=20240730T102822
[2024-07-30T10:28:23.086+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-30T10:28:23.138+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-30T10:28:23.169+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
